locale/template.txt

```diff
@@ -6,30 +6,6 @@ Wind Shield=
 Car battery=
 BoatA=
 Rowboat=
-Black car=
-Blue car=
-Brown car=
-Cyan car=
-Dark green car=
-Dark grey car=
-Green car=
-Grey car=
-Hot Rod car=
-Magenta car=
-Nyan Ride car=
-Oerkki Bliss car=
-Orange car=
-Pink car=
-Red car=
-Road Master car=
-Violet car=
-White car=
-Yellow car=
-Blue hovercraft=
-Green hovercraft=
-Red hovercraft=
-Yellow hovercraft=
-Blue Mesecar=
-Pink Mesecar=
-Purple Mesecar=
-Yellow Mesecar=
+@1 Car=
+@1 Hovercraft=
+@1 Mesecar=
```

Comment:
I am sure this will not translate well in some languages where the adjective (blue, white, green) may depend on the gender of the noun.